R/data-gather-clean.R

```diff
@@ -24,7 +26,9 @@ get_submissions <- function(syn, group, statuses) {
       )
     })
   )
+  
   if (all(is.null(unlist(json_file_paths)))) {
+    print(json_file_paths)
```

Comment:
Are you using the printed messages (this one and others below) for testing only?

Reply:
Thanks Dan, I have removed this now; I'll see how the checks pass.